src/services/NotifService.ts

```diff
@@ -0,0 +1,49 @@
+import { Service } from 'typedi';
+
+import { ExpoPushMessage, PushTicket } from '../types';
+// import { uploadImage } from '../utils/Requests';
+import { Expo } from 'expo-server-sdk';
+const expoServer = new Expo({ accessToken: 'lohuh5OtdO1KS4I_kJGdE0PDtnxgV4GEpTiJAIoZ' });
```

Comment:
the access token looks like it should be in .env

Reply:
yeah it should, my bad i forgot to remove it